manager/state/raft/raft.go

```diff
@@ -394,6 +397,16 @@ func (n *Node) Run(ctx context.Context) error {
 				}
 			}
 
+			n.once.Do(func() {
```

Comment:
Not sure why this has to be in `Run`. Can't we do it right after we load the WAL?